### ISSUE TITLE
Allow team profiles

### ISSUE THREAD

**Issue by pauldw** [pauldw/inflatinator-firmware#9]
Some times, for example, the seabed team is using the same inflator as the ship building team. This means they have to enter the values for their item before they can start inflating. If there was a fast way to switch between different settings ("profiles") this task would be easier.

This is almost like the settings following the person around, rather than sticking with the inflator.

Drew doesn't think this is a super high priority issue because it's tricky to implement and can be worked around effectively.

**Comment by airigami** [comment]
I agree that this is easy to work around since you're really just setting two values, and those values will vary depending on your regulator settings. But a more complex profile for certain shapes/sizes would be nice. Say you're making a dome. It would be nice to enter the height and diameter of the dome, size of balloon, and have it spit out the right sizes of balloons for each row. Status display would show shape and what row you're on.

Mathematically simple to do that, but probably more complex to figure out the right interface design to make it easy to control. Simpler might be to let someone enter a table of values ahead of time that we step through. 

This would be nice, but low priority right now. 

Larry Moss
Airigami.com: The fine art of folding air.

**Comment by DrewRipley** [comment]
This feature needs more thought on how it will work as a whole. The potential is excellent.

**Comment by pauldw** [comment]
You can also hook a laptop up to the Inflatinator's USB port and control the unit from the laptop.
